CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 - [v1.3.0-alpha.3](#v130-alpha3)
   - [Downloads](#downloads)
+- [v1.3.0-alpha.3](#v130-alpha3)
```

Comment:
This was fixed in https://github.com/kubernetes/kubernetes/commit/39c98a8d6b26873b478321bd8b89ee6fe49e1973